CRM/Profile/Form.php

```diff
@@ -689,8 +689,9 @@ public function setDefaultsValues() {
    * Build the form object.
    *
    */
-  public function buildQuickForm() {
+  public function buildQuickForm(): void {
     $this->add('hidden', 'gid', $this->_gid);
+    $this->assign('deleteRecord', $this->isDeleteMode());
```

Comment:
```suggestion
    $this->assign('deleteRecord', $this->_recordExists);
```

Reply:
I figured that part didn't matter because it would bounce & never display the template if the record doesn't exist

Reply:
Ok I see.